src/stores/root.ts

```diff
@@ -1629,6 +1636,21 @@ export class RootStore {
 					},
 				],
 			},
+			{
+				poolId: '692',
```

Comment:
```suggestion
				poolId: '693',
```
Recommend switching to pool 693 as it has a much lower swap fee.

Reply:
Sure, that's a good idea.